src/containers/SearchPage/components/form/SearchContentForm.tsx

```diff
@@ -49,8 +49,7 @@ export interface SearchState extends Record<string, string | boolean | undefined
   includeOtherStatuses: boolean;
   query: string;
   users: string;
-  // This field is called `lang` instead of `language` to NOT match with tag in `SearchTagGroup.tsx`
```

Comment:
Nå matcher disse to. Er det et problem? @jnatten

Reply:
Nei, no som språk er et valgbart felt kan kommentaren slettes.